torch/distributions/cauchy.py

```diff
@@ -0,0 +1,47 @@
+from numbers import Number
+
+import math
+
+import torch
+from torch.distributions import Distribution
+from torch.distributions.utils import broadcast_all
+
+
+class Cauchy(Distribution):
+    r"""
+    Samples from a Cauchy (Lorentz) distribution. The distribution of the ratio of
+    independent normally distributed random variables with means `0` follows a
```

Comment:
Is this "follows" or "roughly follows"? IIRC this distribution was approximate.

Reply:
I thought it wasn't an approximation; there don't seem to be any assumptions -
 http://mathworld.wolfram.com/NormalRatioDistribution.html, but let me know if I missed something.

Reply:
Oh nice, you're right. I was just confused by the wording of http://docs.pyro.ai/primitive_dist.html#module-pyro.distributions.cauchy

Reply:
Yeah, I am not sure about that either. :)  We should change it too, or deprecate it soon.